test/auth_cluster/test_auth_v2_migration.py

```diff
@@ -192,8 +192,7 @@ async def test_auth_v2_during_recovery(manager: ManagerClient):
 
     logging.info(f"Restarting hosts {hosts} in recovery mode")
     await asyncio.gather(*(enter_recovery_state(cql, h) for h in hosts))
-    for srv in servers:
-        await restart(manager, srv)
+    await manager.rolling_restart(servers)
```

Comment:
Perhaps with rolling restarts `reconnect_driver` should not be necessary, the driver should play along nicely. But you never know with the driver...